### ISSUE TITLE
Passing two arguments to cy.its() should throw an err to instead use cy.invoke()

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#147]
asserting off of a value on an attr in `cy.its()` is returns a function a Function instead of the actual value, need to throw error to the user that they really want to use `cy.invoke()`.

**Example Code**
```
cy.get("img").its("attr", "src").should("match", /logo/)
```

**Example Error**
```
expected: [Function] to match /logo/
```

<img width="447" alt="screen shot 2016-05-19 at 12 13 44 pm" src="https://cloud.githubusercontent.com/assets/1271364/15400863/26276f96-1dbb-11e6-85eb-45e862341845.png">

**Example Code**
```
cy.get("a").its("attr", "data-js").should("eq", "edit-company")
```

**Example Error**
```
expected: [Function] to eq "edit-company"
```

**Comment by jennifer-mann** [comment]
Fixed in `0.16.1`

**Comment by brian-mann** [comment]
Fixed in [`0.16.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0161-05222016).
